src/pinning.js

```diff
@@ -443,8 +443,12 @@ class Pinning {
     // assuming address is root store
     const entries = this.openDBs[address].db.iterator({ limit: -1 }).collect()
     // Filter for address-links, get CID, and get to pin it
-    entries.filter(e => e.payload.value.type === 'address-link')
-      .map(e => { this.ipfs.dag.get(e.payload.value.data) })
+    const filter = e => e.payload.value.type === 'address-link' || e.payload.value.type === 'auth-data'
+    entries.filter(filter).map(async e => {
```

Comment:
Would a `forEach` make it more clear that this is an operation instead of a transformation?

Reply:
Good point! I'll update!